dts/bindings/iio/adc/atmel,sam-afec.yaml

```diff
@@ -20,3 +20,9 @@ properties:
       type: int
       description: peripheral ID
       category: required
+
+    "#io-channel-cells":
+      const: 1
+
+"#cells":
+    - input
```

Comment:
Where can I find out what `#cells` means and the effect of having it in the binding?  How is it associated with `#io-channel-cells`?

Reply:
It's explained in https://docs.zephyrproject.org/latest/guides/dts/index.html#bindings (I added a bunch of docmentation to it recently).

That section includes `dts/bindings/binding-template.yaml`, which has the explanation (near the end).

A gnarly thing about our binding format that you can only have a single `#cells`. I want to fix it later, so that you'd say e.g.

```
io-channel-cells:
    - input
```

instead (skipping the `#` is deliberate, because it's kinda awkward).